examples/TaskDynamicsExample.cpp

```diff
@@ -1,33 +1,3 @@
-/* Copyright 2017-2020 CNRS-AIST JRL and CNRS-UM LIRMM
```

Comment:
This is needed (only the full reproduction of the license text is not, sorry if that was not clear in my original comment)

Reply:
Fixed in b8863d5 + added the notice to all files in tests/